src/lib.rs

```diff
@@ -276,8 +283,19 @@ impl Module {
         req: &Parts,
         body: Vec<u8>,
         client_addr: SocketAddr,
+        cache_config_path: String,
     ) -> Result<Response<Body>, anyhow::Error> {
-        let store = Store::default();
+        let start_time = Instant::now();
+        let store = match std::fs::canonicalize(cache_config_path) {
+            Ok(p) => {
+                let mut engine_config = Config::default();
+                engine_config.cache_config_load(p)?;
```

Comment:
This has to load the configuration file on each request - this is not ideal, but the time required to do so is minimal.
We could explore some Wasmtime API changes to allow the easier reuse of the cache configuration.